src/components/Notifications/SilenceNotificationsList.tsx

```diff
@@ -57,50 +87,11 @@ const silenceNotificationListColumns: MRT_ColumnDef<NotificationSilenceItemApiRe
         );
       }
     },
-    {
-      header: "Duration",
-      size: 50,
-      Cell: ({ row }) => {
-        const from = row.original.from;
-        const until = row.original.until;
-        const isExpired = dayjs(until).isBefore(dayjs());
-
-        return (
-          <span>
-            <Age
-              className={clsx(isExpired && "line-through")}
-              from={from}
-              to={until}
-            ></Age>
-            {isExpired && <span className="pl-1 text-red-500">Expired</span>}
-          </span>
-        );
-      }
-    },
     {
       header: "Source",
       accessorKey: "source",
       size: 50
     },
-
-    {
-      header: "Filter",
```

Comment:
We seem to have dropped this ?

Reply:
With the name and description in place, I thought we could drop this.
The table appears cleaner. And most silences, I think, will just have the resource id attached without any filter.

Reply:
Not necessarily - long term silences e.g. non-prod, very noisy alerts, etc... would  have filters and potentially no resource - I think we make it a Resource or Filter column

Reply:
We also miss any error messages on the filter

Reply:
> We also miss any error messages on the filter

The error messages were moved to the name column

![image](https://github.com/user-attachments/assets/85f34dbf-ac2f-4e7d-82d3-c30ed2d87482)